engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicCustomActionsProviderFactory.scala

```diff
@@ -0,0 +1,26 @@
+package pl.touk.nussknacker.engine.management.periodic
+
+import pl.touk.nussknacker.engine.api.deployment._
+import pl.touk.nussknacker.engine.management.periodic.db.PeriodicProcessesRepository
+
+import scala.concurrent.Future
+
+trait PeriodicCustomActionsProviderFactory {
```

Comment:
This pattern could be reused for other `DeploymentManager`s. What do you think about renaming/moving it to `pl.touk.nussknacker.engine.management.CustomActionsProviderFactory` ?

Reply:
But, the point is to have access to `PeriodicProcessesRepository` and `PeriodicProcessService`, cause some implementations of custom actions may need that